### PR TITLE
Add a Czech locale

### PR DESCRIPTION
**Pull request by eatyourgreens**
Staging branch URL: https://pr-5305.pfe-preview.zooniverse.org

Adds a Czech locale (from @yshish).

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Work-in-progress until we have the actual Czech translation in the new language file.

**Comment by eatyourgreens** [comment]
@yshish you can preview your changes on our staging server. Add `language=cs` to view any staging project in Czech eg. https://pr-5305.pfe-preview.zooniverse.org/projects/brooke/i-fancy-cats?language=cs

To view production projects, like Whales As Indivudals, add `env=production` too eg.
https://pr-5305.pfe-preview.zooniverse.org/projects/tedcheese/whales-as-individuals/?language=cs&env=production

**Comment by eatyourgreens** [comment]
@yshish I've added your latest changes. They should appear on the staging site once Jenkins deploys them in the automated checks here.

**Comment by mcbouslog** [comment]
@yshish this is looking good to me, no rush at all, but if you've finished the translation we'll go ahead and merge this, let us know, thanks!

**Comment by yshish** [comment]
> @yshish this is looking good to me, no rush at all, but if you've finished the translation we'll go ahead and merge this, let us know, thanks!

Thanks, but not yet please. There are so many typos and nonsense translations (when I didn't know the context). Needs a lot of work yet. I'll surely let you know as soon as I'm happy with them :)

File: app/locales/cs.js [hunk 1]
```diff
@@ -0,0 +1,1008 @@
...
+      },
+      joinIn: 'Zapojit se',
+      learnMore: 'Dozvědět se více',
+      getStarted: 'Začít',
+      workflowAssignment: 'Otevřela se ti další úroveň s úkoly - %(workflowDisplayName)s',
+      visitLink: 'Navštivte projekt',
+      links: 'Externí odkazy k projektu'
+    }
+  },
+  organization: {ň
```
Comment (eatyourgreens) [inline app/locales/cs.js]:
```suggestion
  organization: {
```

**Comment by eatyourgreens** [comment]
@mcbouslog this should be ready now.

**Pull request merged**